README.md

```diff
@@ -14,23 +14,21 @@ pgvecto.rs is a Postgres extension that provides vector similarity search functi
 ## Why use pgvecto.rs
 
 - 💃 **Easy to use**: pgvecto.rs is a Postgres extension, which means that you can use it directly within your existing database. This makes it easy to integrate into your existing workflows and applications.
+- 🔗 **Blazing fast indexing**: pgvecto.rs is fast not only at index creation, but also at insertion with an index. With async creation support, enjoy your search immediately.
```

Comment:
I think "fast indexing" and "async indexing" is different...

Reply:
fast indexing is comparing to pgvector's single thread indexing

Reply:
> not only at index creation, but also at insertion with an index

it is confusing

Reply:
Do you have any suggestion?

Reply:
> it is fast